shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -123,6 +122,15 @@ - (instancetype)initWithBridge:(shell::AccessibilityBridge*)bridge uid:(int32_t)
   return self;
 }
 
+- (void)dealloc {
+  _bridge = nullptr;
+  _children.clear();
+  [_parent release];
+  if (_container != nil)
```

Comment:
This `nil` check is unnecessary as calling the release selector on nil is a no-op already.

Reply:
Done.